e2e-chatbot-app-next/README.md

```diff
@@ -126,6 +126,123 @@ so that both you and your app service principal can connect to the database, wit
 
    The app starts on [localhost:3000](http://localhost:3000)
 
+### Database Modes
+
+The application supports two operating modes:
+
+#### Persistent Mode (with Database)
+
+This is the default mode when database environment variables are configured. In this mode:
+
+- Chat conversations are saved to Postgres/Lakebase
+- Users can access their chat history via the sidebar
+- Conversations persist across sessions
+- The `/api/history` endpoint returns saved chats
+- A database connection is required (POSTGRES_URL or PGDATABASE env vars)
+
+#### Ephemeral Mode (without Database)
```

Comment:
Any docs/steps on how to actually deploy the app in ephemeral mode? I know you figured out how to do it :P. By default the DAB in this repo attempts to create a lakebase instance